app/views/studio_assessments/index.html.erb

```diff
@@ -1,3 +1,4 @@
+<% if false %>
```

Comment:
if we aren't still testing, let's delete the code that isn't used here before merging.